installer/module/streamlit.py

```
@@ -92,6 +92,12 @@ def start_streamlit_servers(
             entrypoint_path = f"/work/{entrypoint}"
             directory_path = os.path.dirname(entrypoint_path)
 
+            if not os.path.exists(directory_path):
+                logger.warning(
+                    f"Skipping Streamlit app {entrypoint_path!r}: directory {directory_path!r} does not exist"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)